src/main/resources/assets/js/application/recruiter/directives/tdprTableDirective.js

```diff
@@ -99,7 +106,7 @@ define(['angular', 'application/recruiter/tdprRecruiterModule'], function (angul
                     if (newValue !== undefined) {
                         if (newValue === null) {
                             scope.startWeekDay = previousStartWeekDay;
-                            daysLength = 7;
+                            daysLength = _days.length;
```

Comment:
I think we decided to not include one-day functionality - the PO said they don't need it